tests/foreman/cli/test_import.py

```diff
@@ -1184,6 +1184,8 @@ def test_negative_reimport_cv(self):
                     'csv-file': files['custom-channels'],
                     'dir': os.path.join(tmp_dir, 'exports/CHANNELS'),
                 })
+                if bz_bug_is_open('1263650'):
```

Comment:
This BZ is in CLOSE - . WONTFIX status, which means it will never be resolved.

In that case it is not the best to skip the if and use it permanently?